packages/db/schema/sqlite.ts

```diff
@@ -318,6 +318,7 @@ export const sections = sqliteTable("section", {
   width: int(),
   height: int(),
   name: text(),
+  options: text().default('{"json": {}}').notNull(), // empty superjson object
```

Comment:
Add new file in definitions package with a exported `emptySuperjsonObject` to use everwhere where we currently specify this. `emptySuperjsonObject` should have value of `superjson.stringify({})`. Don't forget to install the dependency for it